bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/util/ShellyUtils.java

```diff
@@ -268,6 +269,19 @@ public static DateTimeType getTimestamp(String zone, long timestamp) {
         }
     }
 
+    public static String getTimestamp(DateTimeType dt) {
+        return dt.getZonedDateTime().toString().replace('T', ' ').replace('-', '/');
+    }
+
+    public static String convertTimestamp(long ts) {
+        if (ts == 0) {
+            return "";
+        }
+        String time = DateTimeFormatter.ofPattern(DateTimeType.DATE_PATTERN)
+                .format(ZonedDateTime.ofInstant(Instant.ofEpochSecond(ts), ZoneId.systemDefault()));
```

Comment:
You could make the formatter a constant.

Reply:
changed

Reply:
done